pic-api/src/main/kotlin/com/mashup/pic/security/authentication/JwtAuthentication.kt

```diff
@@ -0,0 +1,42 @@
+package com.mashup.pic.security.authentication
+
+import com.mashup.pic.domain.user.UserRole
+import org.springframework.security.core.Authentication
+import org.springframework.security.core.GrantedAuthority
+
+class JwtAuthentication(private val userInfo: UserInfo) : Authentication {
+
+    private var authenticated: Boolean = false
+
+    override fun getName(): String {
+        return userInfo.nickname
+    }
+
+    override fun getAuthorities(): MutableCollection<out GrantedAuthority> {
+        return userInfo.roles.map(this::convertUserRoleToGrantedAuthority).toMutableList()
+    }
```

Comment:
```suggestion
   override fun getAuthorities(): Collection<GrantedAuthority> {
        return userInfo.roles.map(::convertUserRoleToGrantedAuthority)
    }
```
굳이 Mutable로 반환할 필요는 없을것 같아요